docs/source/torch.compiler_ipex.rst

```diff
@@ -0,0 +1,163 @@
+Intel® Extension for PyTorch* Backend
+=====================================
+
+To work better with `torch.compile`, Intel® Extension for PyTorch* implements a backend `ipex`. It targets to improve hardware resource usage efficiency on Intel platforms for better performance.
+
+Currently the `ipex` backend is implemented with two compilation paths: `TorchScript` path, and `TorchInductor` path. The former path uses `torch.jit.trace` and `torch.jit.freeze` to compile models. The latter path uses `inductor` with further customizations designed in Intel® Extension for PyTorch* for the model compilation. While the `inductor` backend is getting mature, the `TorchScript` path is used as the default path for the `ipex` backend. An API, `ipex._set_compiler_backend("inductor")`, is exposed to turn on the experimental TorchInductor path.
```

Comment:
Suggest not to call out the private API `ipex._set_compiler_backend("inductor")` and the implementation details (`inductor` backend or `TorchScript` backend).